### PR TITLE
fix: Maintaining Consistency in github-readme

### LINKED ISSUE #1633: [BUG] Maintaining Consistency in
# Summary of Issue 

_The Turing Way_ project is an open-source endeavour that empowers contributors from around the world to leverage their skills, knowledge and expertise to build and expand its content.
_The Turing Way_ guides are evolving with multiple chapters co-developed by individuals from varied backgrounds, who are all passionate about sharing knowledge around data science and research.
To sustain and support this vibrant community, _The Turing Way_ book must remain consistent and accessible as it evolves. 
Although the [style guide](https://the-turing-way.netlify.app/community-handbook/style.html) chapter already outlines guidelines for keeping a consistent style across the book, not all chapters follow these suggestions as they have been written asynchronously by different authors.

We have curated a consistency checklist that chapters in _The Turing Way_ need to pass to help us maintain a consistent style across the book. However, we need your help to push this forward!

For reference, the checklist is discussed in the chapter on [maintaining consistency](https://the-turing-way.netlify.app/community-handbook/consistency.html) in _The Turing Way_. There you can also find useful instructions on how to address these issues

## What needs to be done

This issue focuses on maintaining consistency in [README and project communication chapter](https://the-turing-way.netlify.app/reproducible-research/overview/overview-definitions.html) of the Guide for Collaboration. To help with this, we invite contributors to do the following:

- **File Location**: [book/website/collaboration/github/github-readme.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/collaboration/github/github-readme.md)
- **Latest commit when this issue was created**:  f0da923

#### File Specific Checklist

- [ ] Title Case the header so it is `README and Project Communication`
- [ ] Check for proper grammar and spelling
- [ ] Change label from `(rr-github-readme)=` to `(cl-github-readme)=`

#### How you can help
1. Please pick a consistency check from the checklist above (check it off from the issue description)
2. Comment below what consistency check you would like to work on
3. Work on your contributions locally or on Github
4. Raise a PR for your contributions to be reviewed and integrated in _The Turing Way_ repo!

### Who can help

Anyone who would like to make the _The Turing Way_ as consistent and polished as possible. We especially welcome new contributors to try their hand on this!

For any questions, comments, suggestions, or feedback, please comment below :muscle:

### PR DESCRIPTION
**Pull request by afzal442**
### Summary

Fixes #1633

### List of changes proposed in this PR (pull-request)

- [x] 1. Title Case the header so it is README and Project Communication
- [x] 2.  Check for proper grammar and spelling
- [x] 3.  Change label from (rr-github-readme)= to (cl-github-readme)=

After Proposed changes
Screenshot
c2

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by afzal442** [comment]
Hi @paulowoicho , could you review it?

File: book/website/collaboration/github/github-readme.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-(rr-github-readme)=
-# README and project communication
+(cl-github-readme)=
+# README and Project Communication
 
 README files are the welcome mat for your project.
-They are the first thing new visitors to your project will see and thus are part of a set of really important documents to make potential contributors feel welcome and invite them to get involved.
+They are the first thing where new visitors to your project will see and, thus are part of a set of really important documents to make potential contributors feel welcome and invite them to get involved.
```
Comment (paulowoicho) [inline book/website/collaboration/github/github-readme.md:5]:
```suggestion
They are the first thing new visitors to your project will see and are part of a set of essential documents that make potential contributors feel welcome and invite them to get involved.
```

I ran this sentence through Grammarly and it was flagged as being too wordy. My suggestion is what Grammarly recommends.

**Review by paulowoicho — COMMENTED** [review]
Thanks for your work on this! Just made a little suggestion but it looks good to me!

**Review by paulowoicho — APPROVED** [review]
